usaspending_api/references/management/commands/loadtas.py

```diff
@@ -15,9 +17,17 @@ class Command(BaseCommand):
     logger = logging.getLogger('console')
 
     def add_arguments(self, parser):
-        parser.add_argument('file', nargs=1, help='the file to load')
+        parser.add_argument('location', nargs=1, help='The location of the file to load')
 
     def handle(self, *args, **options):
+        is_remote_file = len(options['location'][0].split('.')) == 1
+        if is_remote_file:
+            s3connection = boto.s3.connect_to_region(os.environ.get('AWS_REGION'))
```

Comment:
I don't know if using an environmental variable or using a django setting (like `CSV_AWS_REGION` or `BULK_DOWNLOAD_AWS_REGION`) would be more useful to those running the script. I can run it by the Ops team if that hasn't been discussed.

Reply:
This was the solution that the Ops team and I decided on, instead of using an env variable or hardcoding the bucket name into the script